### PR TITLE
home screen done

### LINKED ISSUE #8: Initial home screen
# REQUIREMENTS
As user I want to access directly to games after login to select which game play.

# DESIGN
We have to show in the home screen a grid with four cards or as many games we have. The cards represents and directly access to the game. The card could have and image and title.

Home:
_________________________
|                       |                        |
|   Game 1       |   Game 2        |
|________________________
|                        |                       |
|   Game 3       |   Game 4       |
|________________________|

# Acceptance Criteria
List all required acceptance criteria
- [ ] Have minimal one direct access
- [ ] Must be the home screen

### PR DESCRIPTION
**Pull request by alanRazoMen**
Co-authored-by: Alex Silva <alexs@users.noreply.github.com>
Co-authored-by: AlejandroCuevasZepeda <AlejandroCuevasZepeda@users.noreply.github.com>
Co-authored-by: MrCano369x <canorueda2000@gmail.com>

# Description
 we made the main page and the routes to the 4 minigames

  ## Screenshots 
home

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -59,46 +66,107 @@ body {
...
-  background-color: #1ab33c;
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 5px;
+  padding: 10px 20px;
+  background-color: #29c34a;
+  border-bottom: 5px solid #13822b;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 15px;
+}
```
Comment (DiegoSteve) [inline app/assets/stylesheets/application.css]: Sort alphabetically.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -59,46 +66,107 @@ body {
...
-  margin-bottom: 10px;
-  background-color: #1ab33c;
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 5px;
+  padding: 10px 20px;
+  background-color: #29c34a;
+  border-bottom: 5px solid #13822b;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 15px;
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: They have many blocks of styles not sorted alphabetically please order them all

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -59,46 +66,107 @@ body {
...
+.minigames div{
+  border-radius: 10px;
+  width: 300px;
+  height: 200px;
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0px 3px 10px -3px white;
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Same here

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -59,46 +66,107 @@ body {
...
+.minigames div p{
+  position: absolute;
+  font-size: 30px;
+  text-shadow: 1px 1px 1px black, -1px -1px 1px black;
+}
+
+.reward{
+  display: flex;
+  align-items: center;
+  padding: 2px 20px;
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Same here

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -59,46 +66,107 @@ body {
...
 }
+.reward img{
+  margin-left: 10px;
+  width: 40px;
+}
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add a blank line here

File: app/assets/stylesheets/application.css [hunk 6]
```diff
@@ -59,46 +66,107 @@ body {
...
+  padding: 10px 20px;
+  background-color: #29c34a;
+  border-bottom: 5px solid #13822b;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 15px;
+}
+.btn:hover{
+  background-color: #13822b;
 }
```
Comment (DiegoSteve) [inline app/assets/stylesheets/application.css]: Space between class and key

File: app/assets/stylesheets/application.css [hunk 7]
```diff
@@ -59,46 +66,107 @@ body {
...
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 5px;
+  padding: 10px 20px;
+  background-color: #29c34a;
+  border-bottom: 5px solid #13822b;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 15px;
+}
+.btn:hover{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 8]
```diff
@@ -59,46 +66,107 @@ body {
...
+  background-color: #13822b;
 }
-.form input[type="submit"]:hover {
-  background-color: #c3380a;
+
+.btn-red{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 9]
```diff
@@ -59,46 +66,107 @@ body {
...
-  font-family: "Goldman";
-  cursor: pointer;
-  width: 100px;
-  height: 30px;
-  margin-bottom: 10px;
-  background-color: #dc3b0e;
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 5px;
+
+.balance{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 10]
```diff
@@ -59,46 +66,107 @@ body {
...
+  justify-content: flex-end;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  gap: 10px;
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Same here

File: app/assets/stylesheets/application.css [hunk 11]
```diff
@@ -59,46 +66,107 @@ body {
...
+}
+
+.balance img{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 12]
```diff
@@ -59,46 +66,107 @@ body {
...
+  width: 40px;
+}
+
+.w100 {
+  width: 100%;
+}
+.center{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 13]
```diff
@@ -59,46 +66,107 @@ body {
...
+  text-align: center;
+}
+
+.container{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 14]
```diff
@@ -59,46 +66,107 @@ body {
...
+  padding: 0px 30px;
+}
+
+.minigames{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 15]
```diff
@@ -59,46 +66,107 @@ body {
...
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+.minigames div{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 16]
```diff
@@ -59,46 +66,107 @@ body {
...
+  height: 200px;
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0px 3px 10px -3px white;
+}
+
+.minigames div:hover{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 17]
```diff
@@ -59,46 +66,107 @@ body {
...
+  top: -5px;
+}
+
+.minigames div img{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

File: app/assets/stylesheets/application.css [hunk 18]
```diff
@@ -59,46 +66,107 @@ body {
...
+  width: 100%;
+  filter: brightness(0.7);
+}
+
+.minigames div p{
```
Comment (Frcomparan) [inline app/assets/stylesheets/application.css]: Add one space between the tag and the curly brace

**Review by Frcomparan — CHANGES_REQUESTED** [review]
Looks good, just some comments

**Review by Frcomparan — APPROVED** [review]
LGTM

**Review by DiegoSteve — APPROVED** [review]
LGBT

**Review by alexs — APPROVED** [review]
LGTM!

**Pull request merged**